sdks/python/src/lilypad/_internal/otel/anthropic.py

```diff
@@ -297,13 +299,13 @@ def _process_chunk(
 
 
 def instrument_anthropic(
-    client: AnthropicClient,
+    client: _AnthropicClient,
```

Comment:
Isn't it better to use public rather than private for the argument type of public functions?